domain-management/src/main/java/org/jboss/as/domain/management/security/LdapGroupSearcherFactory.java

```diff
@@ -338,6 +342,15 @@ private PrincipalToGroupSearcher(final String groupAttribute, final String group
                             if (skipMissingGroups == false) {
                                 throw e;
                             }
+                        } catch (PartialResultException e) {
+                            Hashtable<?, ?> env = null;
+                            if (dirContext != null && (env = dirContext.getEnvironment()) != null && IGNORE.equals(env.get(Context.REFERRAL))) {
+                                /* We can ignore the PartialResultException if we asked for ignoring the referrals
+                                 * - see WFCORE-2341 */
+                                SECURITY_LOGGER.tracef("Ignoring a failed referral query roleNameAttrName: %s", e.getMessage());
```

Comment:
I would suggest here the trace logging should actually log the complete exception, not sure if this is one of those cases but in many cases you can find a message has been logged when the underlying cause had the useful information.